R/pkg/R/DataFrame.R

```diff
@@ -1357,17 +1357,46 @@ setMethod("mutate",
           function(.data, ...) {
             x <- .data
             cols <- list(...)
-            stopifnot(length(cols) > 0)
+            if (length(cols) <= 0) {
+              return(x)
+            }
+
             stopifnot(class(cols[[1]]) == "Column")
+
+            # Check if there is any duplicated column name in the DataFrame
+            dfCols <- columns(x)
+            if (length(unique(dfCols)) != length(dfCols)) {
```

Comment:
perhaps add that in doc or in the error message to say something like "Error: cannot mutate DataFrame with duplicated column names"